cypress/support/scenarios/yves/locale-scenario.ts

```diff
@@ -0,0 +1,28 @@
+import { autoWired } from '@utils';
+import { injectable } from 'inversify';
+
+@injectable()
+@autoWired
+export class LocaleScenario {
+  getAvailableLocales = (): Cypress.Chainable => {
+    return cy.get('[data-qa="language-selector"] option').then((options) => {
```

Comment:
should be in in a page class + repository

Reply:
In scenarios, such relation to CSS selectors is forbidden.